### PR TITLE
Fix space in gas turbine tooltip

### PR DESCRIPTION
**Pull request by the-emerald**
image

### CODE REVIEW

**Review by Connor-Colenso — CHANGES_REQUESTED** [review]
Can we get a formatNumbers around those values while here?

**Comment by the-emerald** [comment]
I'll be honest I have no clue how to do that, I just know enough to find the string and add a space 😛 - searched around the repo, couldn't find a function called `formatNumbers`.

**Comment by miozune** [comment]
it's at `GT_Utility.formatNumbers`

**Comment by Dream-Master** [comment]
@Connor-Colenso can you help here ?

File: src/main/java/common/tileentities/GTMTE_SOFuelCellMK1.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ protected GT_Multiblock_Tooltip_Builder createTooltip() {
         tt.addMachineType("Gas Turbine")
                 .addInfo("Oxidizes gas fuels to generate electricity without polluting the environment")
                 .addInfo(
-                        "Consumes up to" + (EU_PER_TICK * 20)
+                        "Consumes up to " + (EU_PER_TICK * 20)
```
Comment (Connor-Colenso) [inline src/main/java/common/tileentities/GTMTE_SOFuelCellMK1.java]: GT_Utility.formatNumber(EU_PER_TICK * 20)

File: src/main/java/common/tileentities/GTMTE_SOFuelCellMK2.java [hunk 2]
```diff
@@ -83,7 +83,7 @@ protected GT_Multiblock_Tooltip_Builder createTooltip() {
         tt.addMachineType("Gas Turbine")
                 .addInfo("Oxidizes gas fuels to generate electricity without polluting the environment")
                 .addInfo(
-                        "Consumes up to" + (EU_PER_TICK * 20)
+                        "Consumes up to " + (EU_PER_TICK * 20)
```
Comment (Connor-Colenso) [inline src/main/java/common/tileentities/GTMTE_SOFuelCellMK2.java]: GT_Utility.formatNumber(EU_PER_TICK * 20)

**Review by Connor-Colenso — CHANGES_REQUESTED** [review]

**Comment by Dream-Master** [comment]
@the-emerald your turn now

**Review by Glease — APPROVED** [review]
seems ok

**Pull request merged**